src/common/localization/lang.ts

```diff
@@ -0,0 +1,15 @@
+import { addTranslation as add } from './localization';
```

Comment:
I would place even these translations to their own files, if we're doing it to avoid merge conflicts.